src/pages/kimkia/Login/Login.js

```diff
@@ -0,0 +1,78 @@
+import './Login.scss';
+import React, { useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+
+function Login() {
+  const navigate = useNavigate();
+  const [IdValue, setIdvalue] = useState('');
+  const [PWValue, setPWvalue] = useState('');
```

Comment:
```suggestion
  const [idValue, setIdValue] = useState('');
  const [pwValue, setPwValue] = useState('');
```
첫글자를 대문자로 하는 것은 컴포넌트의 네이밍 컨벤션입니다
위와 같이 변경해주세요